CHANGELOG.md

```diff
@@ -1,3 +1,77 @@
+## Release 1.7.0
+
+Highlights:
+- Pod log view: add all container logs
+- Add all Kubernetes namespaces
```

Comment:
Meaning that the namespace selector will now show all namespaces, including empty ones? Deserves clarification.